### PR TITLE
#163503730 user create account 

### PR DESCRIPTION
**Pull request by samorahno**
#### What does this PR do?
This PR gives user the privilege to create new account
#### Description of Task to be completed?
- user can create a new account
#### How should this be manually tested?
By cloning the [Gith Repo](https://samorahno.github.io/Politico/UI) 
use postman to test the endpoint ```POST /auth/signup```
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163503730 
#### Screenshots (if appropriate)
image

#### Questions:
None

### CODE REVIEW

File: server/controllers/UserAuthController.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+      try {
+        const { rows } = await dba.query(createQuery, values);
+        const { id, email, isadmin } = rows[0];
+        const token = userAuthHelper.generateToken({ id, email, isadmin });
+        return res.status(201).header('x-auth-token', token).json({
+          status: 201,
+          data: [{
+            token,
+            user: rows[0],
```
Comment (andela-gike) [inline server/controllers/UserAuthController.js:49]: What is in your user object?
It is clear you are returning your user password which is very insecure

**Review by andela-gike — CHANGES_REQUESTED** [review]

**Comment by andela-gike** [comment]
Your tests are dropping again

File: server/controllers/UserAuthController.js (hunk 1 above)
Comment (samorahno) [reply server/controllers/UserAuthController.js:49]: yes, thanks i also noticed and i corrected it on the branch am working on.

**Comment by samorahno** [comment]
> Your tests are dropping again

This will be fixed when i work on the tests on another branch. i still have to refactor the test

**Pull request closed without merging**